pkg/cluster/resources_test.go

```diff
@@ -62,10 +62,35 @@ func TestConnectionPoolerCreationAndDeletion(t *testing.T) {
 		t.Errorf("%s: Connection pooler service is empty", testName)
 	}
 
-	err = cluster.deleteConnectionPooler()
+	err = cluster.deleteConnectionPooler(Master)
 	if err != nil {
 		t.Errorf("%s: Cannot delete connection pooler, %s", testName, err)
 	}
+
+	//Check if Replica connection pooler can be create and deleted successfully
```

Comment:
Maybe it makes sense to move out replica pooler test into a separate test function.

Reply:
This is also still actual. To clarify, I've meant different initial condition for the test case framed into a separate function.